src/tests/client.ts

```diff
@@ -329,6 +329,51 @@ it('should not send the complete message if the socket is not open', async () =>
   await sub.waitForComplete();
 });
 
+it('should use a custom json reviver and replacer function', async () => {
+  const { url } = await startTServer({
+    replacer: (_, value) => {
+      if (typeof value == 'string') {
+        return value.toUpperCase();
+      }
+      return value;
+    },
+    reviver: (_, value) => {
+      if (typeof value == 'string') {
+        return value.toLowerCase();
+      }
+      return value;
+    },
+  });
+
+  const client = createClient({
+    url,
+    retryAttempts: 0,
+    onNonLazyError: noop,
+    replacer: (_, value) => {
+      if (typeof value == 'string') {
+        return value.toUpperCase();
+      }
+      return value;
+    },
+    reviver: (_, value) => {
+      if (typeof value == 'string') {
+        return value.toLowerCase();
```

Comment:
[The GraphQLSchema `value` field returns the lowercase `"value"`.](https://github.com/gadget-inc/graphql-ws/blob/73bed5e780f1f14cbb117cdb63091f66d77755c8/src/tests/fixtures/simple.ts#L31) This test has no effect on the result, consider maybe using `toUpperCase()` to visualise the alteration.